### PR TITLE
allow unicode in column_legend_fine_print (report builder) fogbugz ca…

### PR DESCRIPTION
**Pull request by javierwilson**
…se #248552

@benrudolph @snopoke

### CODE REVIEW

File: corehq/apps/userreports/reports/builder/forms.py [hunk 1]
```diff
@@ -1188,7 +1188,7 @@ class ConfigureListReportForm(ConfigureNewReportBase):
         widget=forms.HiddenInput,
         error_messages={"required": ugettext_lazy("At least one column is required")},
     )
-    column_legend_fine_print = ugettext_noop("Add columns to your report to display information from cases or form submissions. You may rearrange the order of the columns by dragging the arrows next to the column.")
+    column_legend_fine_print = ugettext_noop(u"Add columns to your report to display information from cases or form submissions. You may rearrange the order of the columns by dragging the arrows next to the column.")
```
Comment (dimagimon) [inline corehq/apps/userreports/reports/builder/forms.py]: E501 line too long (216 > 115 characters)

File: corehq/apps/userreports/reports/builder/forms.py [hunk 2]
```diff
@@ -1284,7 +1284,7 @@ def _report_aggregation_cols(self):
 
 class ConfigureTableReportForm(ConfigureListReportForm, ConfigureBarChartReportForm):
     report_type = 'table'
-    column_legend_fine_print = ugettext_noop('Add columns for this report to aggregate. Each property you add will create a column for every value of that property.  For example, if you add a column for a yes or no question, the report will show a column for "yes" and a column for "no."')
+    column_legend_fine_print = ugettext_noop(u'Add columns for this report to aggregate. Each property you add will create a column for every value of that property.  For example, if you add a column for a yes or no question, the report will show a column for "yes" and a column for "no."')
```
Comment (dimagimon) [inline corehq/apps/userreports/reports/builder/forms.py]: E501 line too long (290 > 115 characters)

File: corehq/apps/userreports/reports/builder/forms.py [hunk 3]
```diff
@@ -1380,7 +1380,7 @@ def _report_aggregation_cols(self):
 class ConfigureWorkerReportForm(ConfigureTableReportForm):
     # This is a ConfigureTableReportForm, but with a predetermined aggregation
     report_type = 'worker'
-    column_legend_fine_print = ugettext_noop('Add columns for this report to aggregate. Each property you add will create a column for every value of that property. For example, if you add a column for a yes or no question, the report will show a column for "yes" and a column for "no".')
+    column_legend_fine_print = ugettext_noop(u'Add columns for this report to aggregate. Each property you add will create a column for every value of that property. For example, if you add a column for a yes or no question, the report will show a column for "yes" and a column for "no".')
```
Comment (dimagimon) [inline corehq/apps/userreports/reports/builder/forms.py]: E501 line too long (289 > 115 characters)

**Comment by snopoke** [comment]
Just to be clear, if these strings (which have no unicode characters in them) get translated to strings which do have unicode characters in them then it fails?

**Comment by benrudolph** [comment]
We were having trouble getting his local setup to translate (fixed now, it was due to cookies, nice find javier). This was more of a "this error usually occurs when byte strings get interpolated with unicode strings" and thought of this as just a general good change in case any of these end up being interpolated with a report name at some point. @javierwilson now that the translations are working, I think it probably makes sense to see if we can reproduce the unicode error

**Comment by javierwilson** [comment]
The actual fix is in the second commit https://github.com/dimagi/commcare-hq/pull/15214/commits/de5727927c54a02b2c09ae74c982eb9faf454f71
the _legend function should return unicode.

**Review by snopoke — APPROVED** [review]

**Pull request merged**